### PR TITLE
Add API count of an API category

### PR DESCRIPTION
**Pull request by thisaltennakoon**
Added API count to the API Categories GET API call in the Admin Side

### CODE REVIEW

File: admin/admin-domain-service/ballerina/apiCategoryImpl.bal [hunk 1]
```diff
@@ -46,8 +46,10 @@ isolated function getAllCategoryList(commons:Organization org) returns APICatego
             foreach APICategory apiCategory in categories {
                 //TODO:(Sampath) need to properly retrieve attached api count per category
```
Comment (CrowleyRajapakse) [inline admin/admin-domain-service/ballerina/apiCategoryImpl.bal:47]: let's remove todo

Comment (thisaltennakoon) [reply admin/admin-domain-service/ballerina/apiCategoryImpl.bal]: Fixed in https://github.com/wso2/apk/pull/1165/commits/13704e6c0a3e4cbb81d7c46e36a0395e2a813b6d

File: admin/admin-domain-service/ballerina/apiCategoryDAO.bal [hunk 2]
```diff
@@ -129,3 +129,19 @@ isolated function deleteAPICategoryDAO(string id, string org) returns commons:AP
...
+    } else {
+        sql:ParameterizedQuery query = `SELECT COUNT(api_id) FROM API WHERE categories?${apiCategory}`;
+        int|sql:Error result = dbClient->queryRow(query);
+        if result is int {
+            return result;
+        } else {
+            log:printError(result.toString());
+            return e909404(result);
+        }
+    }
```
Comment (rdulmina) [inline admin/admin-domain-service/ballerina/apiCategoryDAO.bal:146]:
```suggestion
    }
    sql:ParameterizedQuery query = `SELECT COUNT(api_id) FROM API WHERE categories?${apiCategory}`;
    int|sql:Error result = dbClient->queryRow(query);
    if result is sql:Error {
        log:printError(result.toString());
        return e909404(result);
    } 
    return result;
```

**Review by CrowleyRajapakse — APPROVED** [review]

File: admin/admin-domain-service/ballerina/apiCategoryDAO.bal [hunk 3]
```diff
@@ -129,3 +129,18 @@ isolated function deleteAPICategoryDAO(string id, string org) returns commons:AP
         }
     }
 }
+
+isolated function getAPICountOfAPICategoryDAO(string apiCategory) returns int|commons:APKError {
+    postgresql:Client|error dbClient = getConnection();
+    if dbClient is error {
+        return e909401(dbClient);
+    } else {
```
Comment (rdulmina) [inline admin/admin-domain-service/ballerina/apiCategoryDAO.bal:137]: We can extract out this else block as well

Comment (thisaltennakoon) [reply admin/admin-domain-service/ballerina/apiCategoryDAO.bal]: Fixed in https://github.com/wso2/apk/pull/1165/commits/453160d11adca7c2d3c310c822c32fc50e235f3a

**Review by rdulmina — APPROVED** [review]
LGTM

**Review by Krishanx92 — APPROVED** [review]

**Pull request merged**